addons/mail/static/src/components/attachment_box/attachment_box.xml

```diff
@@ -2,22 +2,16 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.AttachmentBox" owl="1">
-        <t t-if="attachmentBoxView">
+        <t t-if="attachmentBoxView and !attachmentBoxView.chatter.thread.isFileBrowserOpen">
```

Comment:
Not a good condition to have here, instead we should avoid creating the `attachmentBoxView` record when it is not necessary.